### ISSUE TITLE
Add support for ``-h`` flag as well as ``--help``

### ISSUE THREAD

**Issue by tjelvar-olsson** [jic-dtool/dtool-cli#2]
Is there a way to do this with click?

**Comment by woolfeh** [comment]
This is a MWE (adapted from [click doc](http://click.pocoo.org/6/))

```python
import click

# allow '-h' to trigger the help message
CONTEXT_SETTINGS = dict( help_option_names=[ '-h', '--help' ] )

@click.command( context_settings=CONTEXT_SETTINGS )
@click.option('--count', default=1, help='Number of greetings.')
@click.option('--name', prompt='Your name', help='The person to greet.')
def hello(count, name):
    """Simple program that greets NAME for a total of COUNT times."""
    for x in range(count):
        click.echo('Hello %s!' % name)

if __name__ == '__main__':
    hello()

```

**Comment by tjelvar-olsson** [comment]
Fixed in 01d6649
